Kernel/Syscalls/select.cpp

```diff
@@ -148,7 +148,7 @@ int Process::sys$poll(Userspace<const Syscall::SC_poll_params*> user_params)
         return -EFAULT;
 
     // This limit is just a number from the place where numbers come from.
```

Comment:
This comment is no longer true. :)

Reply:
Hmm, yes it is? The numbers (fds) come from `Process`, which contains the line `static const int m_max_open_file_descriptors { FD_SETSIZE };`, or did I misunderstand you?

Reply:
The original meaning of the comment was "this is just a random number that I summoned from the universe's number generator". The new number is meaningful, relevant, and just an overall great choice!

Reply:
Oh! I thought you actually meant something specific with "the place where numbers come from" xD